internal/drand-cli/dkg_cli.go

```diff
@@ -545,11 +545,11 @@ type printModel struct {
 	FinalGroup  string
 }
 
-func convert(entry *drand.DKGEntry) printModel {
+func convert(entry *drand.DKGEntry, checks bool) printModel {
```

Comment:
bool params are ugly - a separate function for rendering checkmarks is clearer imo